tests/tt_eager/python_api_testing/unit_testing/misc/test_optimized_conv_v2.py

```diff
@@ -26,28 +26,33 @@
 
 @skip_for_wormhole_b0()
 @pytest.mark.parametrize(
-    "batch_size, output_channels, input_channels, input_height, input_width, filter_height, filter_width, stride_h, stride_w, pad_h, pad_w, is_1d_systolic",
+    "batch_size, output_channels, input_channels, input_height, input_width, filter_height, filter_width, stride_h, stride_w, pad_h, pad_w, is_1d_systolic, bias, untilize_out",
```

Comment:
Can we add this change in TTNN to ttnn conv API and the `test_conv2d.py`? We want to deprecate the ttlib tests soon. In ttnn, we need support to pass `output_layout`. The file is `ttnn/conv2d.py`.

Reply:
Yes, will do